frog/imports/api/users.js

```diff
@@ -30,7 +30,18 @@ export const getUsername = (user: User, wiki: boolean = false): ?string => {
   }
   return undefined;
 };
-
+/**
+ * Returns the type of the given user if the user is passed or the current user
+ * @param: {User=} user
+ */
+export const userType = (user: User): string => {
+  const selectedUser = getUser(user);
+  // if there is no user logged in then we treat it the same as anonymous
+  if (!selectedUser) return 'Anonymous';
+  else if (selectedUser.isAnonymous) return 'Anonymous';
+  else if (isVerifiedUser({ selectedUser })) return 'Verified';
```

Comment:
BUG: `selectedUser` is not a valid property of type `User`. Why not pass selectedUser directly?